server/mediapublic/exceptions.py

```diff
@@ -0,0 +1,27 @@
+import six
+import json
+import logging
+import cornice.pyramidhook as hooks
+from pyramid import httpexceptions
+from boltons import tbutils
+
+log = logging.getLogger(name="mediapublic.{}".format(__name__))
+
+
+def handle_exceptions(exc, request):
+    # At this stage, the checks done by the validators had been removed because
+    # a new response started (the exception), so we need to do that again.
+    if not isinstance(exc, httpexceptions.HTTPException):
+        e = tbutils.ExceptionInfo.from_current()
```

Comment:
I hate to be "that guy", but I'm usually pretty strongly against single letter variable names ...